Client/Frontend/Brave Rewards/Transfer/WalletTransferExpiredViewController.swift

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import BraveUI
+import Shared
+
+class WalletTransferExpiredViewController: UIViewController, PopoverContentComponent {
+    
+    private let gradientBackgroundView = GradientView(
+        colors: [
+            .init(rgb: 0xA1A8F2),
+            .init(rgb: 0x4C54D2)
+        ],
+        positions: [0.0, 1.0],
+        startPoint: .init(),
+        endPoint: .init(x: 1, y: 1)
+    )
+    
+    private let textLabel = UILabel().then {
+        $0.appearanceTextColor = .white
+        $0.numberOfLines = 0
+        let image = #imageLiteral(resourceName: "warning-triangle").template
+        $0.attributedText = {
+            let imageAttachment = NSTextAttachment().then {
+                $0.image = image
+                var r = CGRect(size: $0.image!.size)
```

Comment:
swiftlint doesn't complain here?

Reply:
Nope! We only have `force_cast` rules in place, not `force_unwrap`